File: ezSCORES/ezSCORES.Model/DTOs/GoalDTO.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text;

namespace ezSCORES.Model.DTOs
{
	public class GoalDTO
	{
		public int Id { get; set; }
		public int? CompetitionTeamPlayerId { get; set; }
		public string? Scorer { get; set; }
		public int ScoredAtMinute { get; set; }
		public bool IsHomeGoal { get; set; }
		public int SequenceNumber { get; set; }

	}
}

File: ezSCORES/UI/ezscores_mobile/lib/models/DTOs/goalDto.dart
import 'package:json_annotation/json_annotation.dart';

part 'goalDto.g.dart';

@JsonSerializable()
class GoalDTO {
	int? id;
  int? competitionTeamPlayerId;
	String? scorer;
	int? scoredAtMinute;
	bool? isHomeGoal;
	int? sequenceNumber;

	GoalDTO(this.id,this.scorer,this.scoredAtMinute,this.isHomeGoal,this.sequenceNumber,);

	factory GoalDTO.fromJson(Map<String, dynamic> json) => _$GoalDTOFromJson(json);

	Map<String, dynamic> toJson() => _$GoalDTOToJson(this);
}
